cibuildwheel/__main__.py

```diff
@@ -367,9 +367,10 @@ def detect_warnings(platform: str, build_options: BuildOptions) -> List[str]:
     for option_name in ['test_command', 'before_build']:
         option_value = getattr(build_options, option_name)
 
-        if option_value:
-            if '{python}' in option_value or '{pip}' in option_value:
-                warnings.append(option_name + ": '{python}' and '{pip}' are no longer needed, and will be removed in a future release. Simply use 'python' or 'pip' instead.")
+        if option_value and (
+            '{python}' in option_value or '{pip}' in option_value
+        ):
```

Comment:
Given the length of the next line, this could just as well be on one line?

Reply:
This doesn't know that we've increased black's line width to 120. This would be the normal way to do it if running with 80/88 line width. It makes it much harder to miss the `()`, too.

```suggestion
        if option_value and ('{python}' in option_value or '{pip}' in option_value):
```